services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/process_message_running_service.py

```diff
@@ -295,17 +295,24 @@ async def _process_stop_event(
             msg.created_at,
             running_service.pricing_unit_cost,
         )
-
-        wallet_total_credits = await credit_transactions_db.sum_wallet_credits(
-            db_engine,
-            product_name=running_service.product_name,
-            wallet_id=running_service.wallet_id,
+        # NOTE: Include_pending_transactions=False will ensure that we do not count the current running transactions.
```

Comment:
TIP: embed the comment within the code so it does not move away in future refactoring
i.e.
```python
 wallet_total_credits_without_pending_transactions = (
            # NOTE: .... add comments here 
            await credit_transactions_db.sum_wallet_credits(
                db_engine,
                product_name=running_service.product_name,
                wallet_id=running_service.wallet_id,
                include_pending_transactions=False,
            )
        )
```